WPFHospitalEditor/Service/EquipmentServerService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using HealthcareBase.Dto;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+
+namespace WPFHospitalEditor.Service
+{
+   public  class EquipmentServerService
+    {
+        public Dictionary<String,EquipmentDto> getEquipmentByRoomId(int roomId)
+        {
+            var client = new RestClient("http://localhost:5290/");
```

Comment:
It is better to create a constant where the url will be defined. Any change in the url would only change that constant instead of changing that data in many more places.

Reply:
fixed